src/scripts/main.js

```diff
@@ -1,7 +1,29 @@
 'use strict';
 
 const pushNotification = (posTop, posRight, title, description, type) => {
-  // write code here
+  const message = document.createElement('div');
+
+  message.className = 'notification';
+  message.classList.add(type);
```

Comment:
Ensure that the `type` parameter corresponds to a valid CSS class. If the `type` does not match any predefined class, it might lead to unexpected styling issues.